arrows/qt/widgets/MetadataView.cxx

```diff
@@ -290,7 +317,6 @@ void MetadataView::updateMetadata(kv::metadata_vector const& mdVec)
 }
 
 //END MetadataView
-
```

Comment:
I'm pretty sure this blank line is supposed to be here. I think this is probably something you deleted while also deleting the blanks between namespace levels?